db/migrate/20200324210540_rename_default_variable_lookup_path.rb

```diff
@@ -0,0 +1,5 @@
+class RenameDefaultVariableLookupPath < ActiveRecord::Migration[5.2]
+  def up
+    Setting.where(:name => 'Default_variables_Lookup_Path').update_all(:name => N_('Default_parameters_Lookup_Path'), :description => N_("Foreman will evaluate host smart class parameters in this order by default"))
```

Comment:
Failures are related - the new setting could already exist at this point (because they are created in app initialization), so if it does we need to update it's value to the value of the old setting and delete the old setting.